zokrates_core/src/semantics.rs

```diff
@@ -3757,15 +3705,9 @@ mod tests {
         .mock()
     }
 
-    /// Helper function to create ((private field a): return)
+    /// Helper function to create def function1(private field a) { return; }
```

Comment:
I didnt name these because the `Function` struct does not hold the name

Reply:
oh I see